### PR TITLE
API-1406 - Add new stringReplace for processSelect

### PR DESCRIPTION
**Pull request by jocteau**
Add the edit mode: `stringReplace` to `processSelect` and also add the possibility of using an array of value for the `Replace` string
Screen Shot 2021-04-28 at 2 13 28 PM
Screen Shot 2021-04-28 at 2 13 24 PM

### CODE REVIEW

File: src/cn-batch-forms.service.js [hunk 1]
```diff
@@ -127,14 +127,33 @@ export function setValue(ffService) {
...
       const withStr = ffService.parseExpression(`__with_${key}`, model).get() || '';
-      const updateVal = replaceStr ?
-        original.get().replace(replaceExp, withStr).trim() :
-        original.get();
-      update.set(updateVal);
+      console.log('-----------------')
+      console.log('replaceStr', replaceStr)
+      console.log('withStr', withStr)
+      console.log('originalVal', originalVal)
+
```
Comment (davezuch) [inline src/cn-batch-forms.service.js]: Did you mean to leave these?

**Review by davezuch — APPROVED** [review]
LGTM.

File: src/cn-batch-forms.service.js (hunk 1 above)
Comment (jocteau) [reply src/cn-batch-forms.service.js]: Good catch, I'll remove them

**Review by matt-morales — COMMENTED** [review]
Well done dude 🥇

**Review by matt-morales — APPROVED** [review]

**Pull request merged**